packages/blockchain-wallet-v4-frontend/src/modals/SimpleBuy/Frequency/index.tsx

```diff
@@ -0,0 +1,78 @@
+import React, { PureComponent } from 'react'
+import { connect, ConnectedProps } from 'react-redux'
+import { bindActionCreators, Dispatch } from 'redux'
+
+import { RemoteDataType, SBPaymentMethodType } from 'blockchain-wallet-v4/src/types'
+import { FrequencyScreen } from 'components/Flyout'
+import { actions } from 'data'
+import { RootState } from 'data/rootReducer'
+import { RecurringBuyPeriods } from 'data/types'
+
+import getData from './selectors'
+
+class Frequency extends PureComponent<Props> {
+  constructor(props) {
+    super(props)
+    this.handleFrequencySelection = this.handleFrequencySelection.bind(this)
```

Comment:
should be able to delete this constructor.  you shouldnt need to bind `this` when you define the function with a fat arrow